frontend/src/views/Pages/Login.vue

```diff
@@ -20,6 +20,12 @@
             <b-card-body class="p-4">
               <div class="text-center text-muted mb-4">
                 <small>{{ $t('login') }}</small>
+              </div>
+                <div class="text-center text-muted mb-4">
+                 {{ $store.state.community_name }}
```

Comment:
Why do we define this so? The name would also be in $store.state.community.name. 
I don't really like names like community_name

Reply:
There were serious problems with me. For some inexplicable reason, it was not possible to retrieve the name, or it was incorrect. 
Since it will come from the ENV or CONFG anyway, there will no longer be any variables at this point, will there?